Automattic/jetpack
# Proposal: Rename AIOWP tracks events so we don't count them as site migrations

Related to https://github.com/Automattic/dotcom-forge/issues/9695

## Proposed changes:

* We've been counting AIOWP as site migrations even though they aren't necessarily, and they don't go through our migration flows, so this throws our data off.
* Rename the events so they more accurately reflect what we're tracking.

### Other information:


## Jetpack product discussion



N/A

## Does this pull request change what data or activity we track or use?



No.

## Testing instructions:






* Visual inspection should be sufficient, but if you want to test the events:

1. Create a new Business site, mark it as a WoA Dev site, and take it Atomic.
2. Rsync the wpcomsh changes on this branch to your WoA site: `jetpack rsync wpcomsh yourtestsite.wordpress.com@sftp.wp.com:htdocs/wp-content/mu-plugins`
3. On your WoA test site, install the AIOWPM plugin: https://wordpress.com/plugins/all-in-one-wp-migration/
4. Activate the plugin and go to All in One WP Migration -> Export
5. Create and download an export file
6. Go to All in One WP Migration -> Import
7. Start the import from the file you created
8. Check Tracks Live for the migration start and end events (may take a few minutes to get there): `wpcom_import_start` and `wpcom_import_done`
9. Ensure the events have a `target_blog_id` eventprop set to your test site's blog ID
